exec_node_handling.c

```diff
@@ -17,7 +17,7 @@ int	other_execution_type(t_tools *tool, t_execcmd *ecmd)
 {
 	if (is_builtin(ecmd->argv[0]))
 		exit(run_builtin(ecmd, tool));
-	if (ft_strncmp(ecmd->argv[0], "minishell", 9) == 0 ||ft_strncmp(ecmd->argv[0], "./minishell", 11) == 0)
+	if (ft_strncmp(ecmd->argv[0], "minishell", 9) == 0 || ft_strncmp(ecmd->argv[0], "./minishell", 11) == 0)
```

Comment:
the checks need to include either the null terminator or make sure that there is a space after minishell because currently it runs minishell if you typein minishellhello or minishellll.... and it prob shouldn't...

Reply:
yes true - we can just have it compare one more index